src/modules/commentUI.js

```diff
@@ -0,0 +1,39 @@
+export default class CommentUI {
+  static addFacts = async (parentElement, facts) => {
+    // clear old entries
+    parentElement.innerHTML = '';
+    // [type, shortcut, author]
```

Comment:
- [ ] The same thing for this line, try not to commit the commented codes so that your code is clean.

Reply:
Implemented